base/shell/explorer/taskswnd.cpp

```diff
@@ -153,6 +153,10 @@ class CHardErrorThread
         if (FAILED_UNEXPECTEDLY(hr))
             return hr;
 
+        hr = pnotification->SetIconInfo(NULL, NULL);
```

Comment:
Please check on Windows, when you get a notification, which tooltip text you have when you point on the notification icon. Iirc it should be the same as the title of the balloon.

Reply:
It looks like it doesn't have a tooltip.

Reply:
On Windows 7 it looks like this:
![win7_harderror_balloon](https://user-images.githubusercontent.com/1969829/36647572-9ac6b830-1a7f-11e8-819a-d699057cdb2e.png)

Reply:
And you consider that a nice result that we should emulate?

Reply:
It's up to you to choose: if you consider it's ugly to have a tooltip in addition to the balloon, then keep your code as it is. Otherwise you know my answer.